### ISSUE TITLE
Incorrect error message when over-using empty string

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#672]
# Bug report

## What's wrong

Right now:

> hooks/post_gen_project.py:0:1: Z226 Found string constant over-use: 

## How is that should be

> hooks/post_gen_project.py:0:1: Z226 Found string constant over-use: ''

I am using `0.10`

**Comment by DjaPy** [comment]
Hey!
I wrote a new test. And test falls.
```
 assert "Z226 Found s... over-use: ''" == 'Z226 Found st...nt over-use: '
E         - Z226 Found string constant over-use: ''
E         ?                                      --
E         + Z226 Found string constant over-use:
```
I add the following rule to the code
```
 def _post_visit(self) -> None:
        for string, usage_count in self._string_constants.items():
            if not len(string):
                string = "''"
            if usage_count > self.options.max_string_usages:
                self.add_violation(OverusedStringViolation(text=string))
```
Then the previous test falls.
How to catch an empty line for a visitor?

**Comment by sobolevn** [comment]
I would simplify your code to:

```python
def _post_visit(self) -> None:
        for string, usage_count in self._string_constants.items():
            if usage_count > self.options.max_string_usages:
                self.add_violation(OverusedStringViolation(text=string or "''"))
```

That's it. Tests should just be fixed. Make sure that `assert_error_text` works.
